test/component_test.js

```diff
@@ -67,7 +67,7 @@ test("Defines the correct loader", function(){
 	myLoader.import(myLoader.configMain).then(function(){
 		return myLoader.import("test/tests/frankenstein.component!");
 	}).then(function(){
-		var template = defines["test/tests/frankenstein.component-template"];
+		var template = defines["test/tests/frankenstein.component-view"];
```

Comment:
@matthewp I imagine that we still want to use `<template>` tags, but do we want this part to be `component-view`?